pcdsdevices/epics_motor.py

```diff
@@ -758,6 +781,47 @@ def get_configuration(self):
         self._setup_and_check_pmgr()
         return self._pm.get_config(self.prefix)
 
+    def get_configuration_values(self, cfgname=None):
+        """
+        Return the current configuration parameters of the motor
+        in the parameter manager
+
+        Parameters
+        ----------
+        cfgname : str
+               The name of the configuration
+
+        Returns
+        _______
```

Comment:
Docs  build expects hyphens
```suggestion
        -------
```